HANDOFF.md

```
@@ -1206,3 +1206,18 @@ it), not a re-derivation of (a).
 4. **Safe PFS Mount on Save**:
    - In `src/opl.c:trySaveConfigHDD()`, ensured `hddLoadSupportModules()` mounts `pfs0:` before writing.
 
+# 31. STEP-201: Fix MX4SIO Game Launching Hang on Game-Side IOP Reset (2026-08-17)
+
+### Root Cause:
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)